src/test/java/net/openhft/chronicle/bytes/BytesTest.java

```diff
@@ -593,7 +593,7 @@ public void testWithLength() {
     public void testAppendBase() {
         assumeFalse(alloc1 == HEX_DUMP);
         @NotNull Bytes<?> b = alloc1.elasticBytes(16);
-        for (long value : new long[]{Long.MIN_VALUE, Integer.MIN_VALUE, ~4, ~2, ~1, ~0, 0, 1, 2, 4, 8, 16, 32, 64, 128, 256, Integer.MAX_VALUE, Long.MAX_VALUE}) {
+        for (long value : new long[]{Long.MIN_VALUE, Integer.MIN_VALUE, -1, 0, 1, Integer.MAX_VALUE, Long.MAX_VALUE}) {
```

Comment:
Why?

Reply:
That's a complete revert of what went by mistake to develop prematurely. Will be reapplied later.